common/app/googleAuth/GoogleAuthFilters.scala

```diff
@@ -15,9 +14,8 @@ object GoogleAuthFilters {
   val LOGIN_ORIGIN_KEY = "loginOriginUrl"
 
   class AuthFilterWithExemptions(loginUrl: FilterExemption, exemptions: Seq[FilterExemption])
-                                (implicit val mat: Materializer, environment: Environment, cryptoConfig: CryptoConfig) extends Filter {
+                                (implicit val mat: Materializer, environment: Environment) extends Filter {
 
```

Comment:
bad merge. We need to create an instance of `AuthCookie` with `cryptoConfig` param